### DISCUSSION TITLE
Connecting QA Retrieval to Prompt and Output Parser

### DISCUSSION THREAD

**Discussion by zapulam** [FlowiseAI/Flowise#2417] [Q&A]
Hello,
So far when creating my Flowise chain I am able to load in a PDF and connect an LLM in order to allow for Q&A on the document, which works great. However I would like to take this one step further and connect prompts and output parsers in order to only retrieve certain information in JSON format. If I can connect the output of my Retrieval QA Chain to another sequential chain I believe this would easily solve the issue but I cannot find a way to do this. Any support is greatly appreciated. I have also provided a screenshot of how things look so far.
 
![ss](https://github.com/FlowiseAI/Flowise/assets/84103628/e3c6f8f4-6158-4d7d-9a1a-8d05369cc18a)

**Comment by Eng-AliKazemi** [comment]
I have the same issue

**Comment by Astriel** [comment]
I've got the same issue, did you managed to find an answer ?

**Comment by Astriel** [comment]
I guess the whole JSON format could be included in the prompt, but I guess there are other ways of doing this.
